src/lib/image/bootstrap/busybox/busybox.c

```diff
@@ -0,0 +1,45 @@
+/* 
```

Comment:
Is there any reason why we would want to port the individual modules into C, or are they fine being left in shell? I was thinking that we might want to port some code into C and therefore need the different module.c files. Your call, I wasn't sure